src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -54,39 +59,95 @@ spec:
     namespace: default
 ```
 
-*AWSCluster* CR object example:
+### `AWSCluster`
 
 ```yaml
 apiVersion: infrastructure.giantswarm.io/v1alpha2
-kind: AWSMachineDeployment
+kind: AWSCluster
 metadata:
+  generation: 1
   labels:
-    aws-operator.giantswarm.io/version: 8.1.1
+    aws-operator.giantswarm.io/version: 8.7.0
     giantswarm.io/cluster: nzr5z
-    giantswarm.io/machine-deployment: pv7ps
-    giantswarm.io/organization: giantswarm
-    release.giantswarm.io/version: 11.0.1
-  name: pv7ps
+    giantswarm.io/organization: acme
+    release.giantswarm.io/version: 11.4.0
+  name: nzr5z
   namespace: default
 spec:
-  nodePool:
-    description: np1
-    machine:
-      dockerVolumeSizeGB: 100
-      kubeletVolumeSizeGB: 100
-    scaling:
-      max: 10
-      min: 3
+  cluster:
+    description: Demo cluster
+    dns:
+      domain: gorilla.eu-central-1.aws.gigantic.io
+    oidc:
+      claims: {}
   provider:
-    availabilityZones:
-    - eu-central-1a
-    worker:
-      instanceType: m4.xlarge
+    credentialSecret:
+      name: credential-default
+      namespace: giantswarm
+    master:
+      availabilityZone: ""
+      instanceType: ""
```

Comment:
We need to migrate this structure here away in the mid term. Should we still show it in the docs?

Reply:
If the CR is valid without the `master` part I am all for removing it in docs.

Reply:
Looks pretty optional to me: https://github.com/giantswarm/apiextensions/blob/master/pkg/apis/infrastructure/v1alpha2/aws_cluster_types.go#L116-L119.